pkg/cmd/helmfile/move/move.go

```diff
@@ -223,8 +223,13 @@ func (o *Options) lazyCreateNamespaceResource(ns string) error {
 }
 
 func (o *Options) moveFilesToClusterOrNamespacesFolder(dir, ns, releaseName, chartName string) error {
+	var err error
 	o.ClusterWide = make(map[string]bool)
-	JXClient, err := jxclient.LazyCreateJXClient(nil)
```

Comment:
Do we need jxclient? Unit tests pass with normal kubeclient, but I am not sure ...